src/pip/_internal/req/req_file.py

```diff
@@ -4,6 +4,7 @@
 
 import optparse
 import os
+import os.path
```

Comment:
I believe this is not needed for modern Python versions (since we already import `os`).

Reply:
(I concur)